Lib/idlelib/idle_test/test_config.py

```diff
@@ -26,6 +28,162 @@ def tearDownModule():
 
 
+class IdleConfParserTest(unittest.TestCase):
+    """Test IdleConfParser works"""
+
+    config = """
+        [one]
+        one = false
+        two = true
+        three = 10
+
+        [two]
+        one = a string
+        two = true
+        three = false
+    """
+
+    def test_get(self):
+        parser = config.IdleConfParser('')
+        parser.read_string(self.config)
+
+        eq = self.assertEqual
+
+        # Test with type
+        eq(parser.Get('one', 'one', type='bool'), False)
+        eq(parser.Get('one', 'two', type='bool'), True)
+        eq(parser.Get('one', 'three', type='int'), 10)
+        eq(parser.Get('two', 'one'), 'a string')
+        eq(parser.Get('two', 'two', type='bool'), True)
+        eq(parser.Get('two', 'three', type='bool'), False)
+
+        # Test without type should fallback to string
+        eq(parser.Get('two', 'two'), 'true')
+        eq(parser.Get('two', 'three'), 'false')
+
+        # If option not exist, should return None, or default
+        eq(parser.Get('not', 'exist'), None)
+        eq(parser.Get('not', 'exist', default='DEFAULT'), 'DEFAULT')
+
+    def test_get_option_list(self):
+        parser = config.IdleConfParser('')
+        parser.read_string(self.config)
+
+        self.assertEqual(parser.GetOptionList('one'), ['one', 'two', 'three'])
+        self.assertEqual(parser.GetOptionList('two'), ['one', 'two', 'three'])
+        self.assertEqual(parser.GetOptionList('not exist'), [])
+
+    def test_load_file(self):
+        # Test configfile 'cfgparser.1' borrow from test_configparser
+        config_path = findfile('cfgparser.1')
+        parser = config.IdleConfParser(config_path)
+        parser.Load()
+
+        self.assertEqual(parser.Get('Foo Bar', 'foo'), 'newbar')
+        self.assertEqual(parser.GetOptionList('Foo Bar'), ['foo'])
+
+
+class IdleUserConfParserTest(unittest.TestCase):
+    """Test IdleUserConfParser works"""
+
+    def new_parser(self, path=''):
+        return config.IdleUserConfParser(path)
+
+    def test_add_section(self):
+        parser = self.new_parser()
+        self.assertEqual(parser.sections(), [])
+
+        # Duplicate section should only add one time.
+        # In normal configparser, it will raise DuplicateError,
+        # IdleParser won't raise it
+        parser.AddSection('Foo')
+        parser.AddSection('Foo')
+        parser.AddSection('Bar')
+        s = parser.sections()
+        s.sort()
+        self.assertEqual(s, ['Bar', 'Foo'])
+
+    def test_remove_empty_sections(self):
+        parser = self.new_parser()
+
+        parser.AddSection('Foo')
+        parser.AddSection('Bar')
+        self.assertEqual(sorted(parser.sections()), ['Bar', 'Foo'])
+        parser.RemoveEmptySections()
+        self.assertEqual(parser.sections(), [])
+
+    def test_is_empty(self):
+        parser = self.new_parser()
+
+        parser.AddSection('Foo')
+        parser.AddSection('Bar')
+        self.assertEqual(parser.IsEmpty(), True)
```

Comment:
`assertTrue()`.

Or `assertIs()` if you want to check that exact bool is returned. But I think this is redundant.